src/main/sensors/gyro.h

```diff
@@ -144,13 +133,15 @@ typedef struct gyroConfig_s {
     int16_t  yaw_spin_threshold;
 
     uint16_t gyroCalibrationDuration;   // Gyro calibration duration in 1/100 second
-    
+
     uint16_t dyn_lpf_gyro_min_hz;
     uint16_t dyn_lpf_gyro_max_hz;
-    uint8_t  dyn_notch_range;            // ignore any FFT bin below this threshold
+
+    uint16_t dyn_notch_max_hz;
```

Comment:
Since you're changing the structure of this (by changing from uint8 to uint16) you must increment the PG version otherwise the parameter group will be corrupted from saved settings. In gyro.c at about line 160 change:
```
PG_REGISTER_WITH_RESET_FN(gyroConfig_t, gyroConfig, PG_GYRO_CONFIG, 7);
```
to
```
PG_REGISTER_WITH_RESET_FN(gyroConfig_t, gyroConfig, PG_GYRO_CONFIG, 8);
```